en/appendices/3-4-migration-guide.rst

```diff
@@ -33,6 +33,8 @@ elements will continue to function until 4.0.0 after which they will be removed.
 
 * The ``Auth.redirect`` session variable is no longer used. Instead a query
   string parameter is used to store the redirect URL.
+* ``AuthComponent`` no longer stores redirect URLs when the unauthorized URL is
+  a ``POST`` action.
```

Comment:
technically also PUT and PATCH I guess - maybe we can say "if not a GET action"?